packages/open-next/src/wrappers/aws-lambda-streaming.ts

```diff
@@ -76,8 +76,6 @@ const handler: WrapperHandler = async (handler, converter) =>
 
       const streamCreator: StreamCreator = {
         writeHeaders: (_prelude) => {
-          _prelude.headers["content-encoding"] = contentEncoding;
```

Comment:
Is it duplicated ?
I think we need to set it here, the compression is done by the wrapper itself

Reply:
<img width="798" alt="image" src="https://github.com/user-attachments/assets/4de1a19b-6bcd-4565-8b15-359ed1e3b7fb">

Check l84

Reply:
:facepalm: My bad